src/Microsoft.AspNet.Mvc.Razor/RazorServiceCollectionExtensions.cs

```diff
@@ -24,5 +25,28 @@ public static class ServiceCollectionExtensions
         {
             services.Configure(setupAction);
         }
+
+        /// <summary>
+        /// Adds an initialization callback for a given <see cref="ITagHelper"/> type.
+        /// </summary>
+        /// <remarks>
+        /// The callback will be invoked on the <see cref="ITagHelper"/> of the specified type before the
```

Comment:
`the <see cref="ITagHelper"/> of the specified type` -> `any <typeparamref name="TTagHelper"> instance` or at least something that aligns better with line 30.